src/app/components/dialogs/baseDialog/baseDialog.component.html

```diff
@@ -1,4 +1,5 @@
 <h1>{{ null != title ? title : '' }}</h1>
+<button class="close" mat-button (click)="onNoClick()">X</button>
```

Comment:
What is the functionality of another close button?

Reply:
I guess it is not really needed, the "back " button is sufficient, also the styling of the button is messed up with the new angular version.
Instead I feel it would be good that when clicking outside the dialog the cancel/back is triggered

Reply:
if you set the 'closeable' in the dialog to true (in dialog.service.ts) when creating it the dialog will close when the user clicks outside of the dialog